drivers/sensor/vl53l0x/vl53l0x.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SENSOR_VL53L0X_H__
+#define __SENSOR_VL53L0X_H__
+
+#include <zephyr/types.h>
+#include <device.h>
+#include "vl53l0x_api.h"
+#include "vl53l0x_platform.h"
+
+#define SYS_LOG_DOMAIN "VL53L0X"
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_SENSOR_LEVEL
+#include <logging/sys_log.h>
+
+#define VL53L0X_I2C_ADDR       0x29
+#define VL53L0X_REG_WHO_AM_I   0xC0
+#define VL53L0X_CHIP_ID        0xEEAA
+#define VL53L0X_SETUP_SIGNAL_LIMIT         (0.1*65536)
+#define VL53L0X_SETUP_SIGMA_LIMIT          (60*65536)
+#define VL53L0X_SETUP_MAX_TIME_FOR_RANGING     33000
+#define VL53L0X_SETUP_PRE_RANGE_VCSEL_PERIOD   18
+#define VL53L0X_SETUP_FINAL_RANGE_VCSEL_PERIOD 14
```

Comment:
Can you reference if possible all these numbers above? (datasheet chapter/section will do)